code/__HELPERS/mobs.dm

```diff
@@ -298,6 +298,8 @@ Proc for attack log creation, because really why not
 	var/endtime = world.time + delay
 	var/starttime = world.time
 	. = 1
+	var/mob/living/L = user
+	var/is_living = istype(L)

```

Comment:
Why not just define IsStun() on /mob returning `TRUE` and override on living? Would avoid doing this.